learners/setup.md

```diff
@@ -2,53 +2,154 @@
 title: Setup
 ---
 
-FIXME: Setup instructions live in this document. Please specify the tools and
-the data sets the Learner needs to have installed.
+The vantage6 has requirements on hardware and software to run its server and node instances. As for the detailed requirements, please refer to the [vantage6 documentation](https://docs.vantage6.ai/en/main/node/requirements.html).
```

Comment:
I would remove *the*. The vantage6 sounds a bit weird. And maybe then reformulate a little bit to: 

> vantage6 has hardware and software requirements in order to run the vantage6 node(s) and/or server.